src/Tizen.Multimedia/MediaTool/MediaPacket.cs

```diff
@@ -656,6 +657,22 @@ public static MediaPacket Create(MediaFormat format)
             return new SimpleMediaPacket(format);
         }
 
+        /// <summary>
+        /// Creates an object of the MediaPacket based on the original MediaPacket and increase ref count of both MediaPacket instances.
```

Comment:
1. "and increase" -> "and increases"
2. I'm a little confused about the word "both", I was expecting that the refcount is attached to underlying native media packet and both C# MediaPacket instances share that native one. If there is only one refcount and it is not possible that "both MediaPacket instances" have a different value of ref count, then please consider changing the description to sth like "and increases the ref count of media packet internal data" / "media packet buffer" / "native media packet" / "native media packet handle". 
3. I'm wondering where we should write "reference counter" instead of short "ref count", at least once, probably here. But the term "ref count" is well known, so it would be ok to keep only that short term.

Reply:
yes, that would be good